### ISSUE TITLE
Cell::TestCase using a concept with collection

### ISSUE THREAD

**Issue by glapworth** [apotonick/cells#380]
When using a `Cell::TestCase` with a concept that includes a collection, the following error occurs:

```
NoMethodError: undefined method `must_have_selector' for #<ActiveSupport::SafeBuffer
```

I am able to produce this using the first `Cell::TestCase` example from the TrailBlazer book (chapter 4):

```
class ThingCellTest < Cell::TestCase
  controller ThingsController

  let(:rails) { Thing::Create.(thing: {name: "Rails"}) }
  let(:trb) { Thing::Create.(thing: {name: "Trailblazer"}) }

  subject { concept("thing/cell", collection: [trb, rails], last: rails).model }

  it do
    subject.must_have_selector ".columns .header a", text: "Rails"
    subject.wont_have_selector ".columns.end .header a", text: "Rails"
    subject.must_have_selector ".columns.end .header a", text: "Trailblazer"
  end
end
```

I'm using Cells 4.0.3, but have also tested against Cells 4.0.0 and here is the corresponding version information for MiniTest et al:

```
initest (5.8.4)
    minitest-capybara (0.7.2)
      capybara (~> 2.2)
      minitest (~> 5.0)
      rake
    minitest-line (0.6.3)
      minitest (~> 5.0)
    minitest-metadata (0.5.3)
      minitest (>= 4.7, < 6.0)
    minitest-rails (2.2.0)
      minitest (~> 5.7)
      railties (~> 4.1)
    minitest-rails-capybara (2.1.1)
      capybara (~> 2.0)
      minitest-capybara (~> 0.7.0)
      minitest-metadata (~> 0.5.0)
      minitest-rails (~> 2.1)
    minitest-spec-rails (5.3.0)
      minitest (~> 5.0)
      rails (>= 4.1)
```

**Comment by tycooon** [comment]
I investigated `Cell::Testing` code a bit, and found out, that capybara feature relies on `ViewModel#to_s` calling `call` method, which is not true when rendering collections, since `render_collection` method always returns `ActiveSupport::SafeBuffer` and not a cell instance.

**Comment by timoschilling** [comment]
I know this issue and working on new collection interface

**Comment by timoschilling** [comment]
Here is a workaround:
```ruby
subject { Capybara.string(concept("thing/cell", collection: [trb, rails], last: rails)) }
```

**Comment by glapworth** [comment]
I have to change that slightly, otherwise the call to the "thing/cell" doesn't work correctly:

`subject { Capybara.string(concept("thing/cell", collection: [trb.model, rails.model], last: rails.model)) }`

However, this code causes the following exception:

```
NoMethodError: undefined method `string' for Cell::Testing::Capybara:Module
    test/concepts/thing/cell_test.rb:9:in `block in <class:ThingCellTest>'
    test/concepts/thing/cell_test.rb:12:in `block in <class:ThingCellTest>'
```

**Comment by timoschilling** [comment]
try:
```ruby
`subject { ::Capybara.string(concept("thing/cell", collection: [trb.model, rails.model], last: rails.model)) }`
```

**Comment by glapworth** [comment]
:+1:  @timoschilling that works, as expected.  Thanks for your help.

**Comment by apotonick** [comment]
Fixed in 4.0.5, with an ugly hack, and awesome again in 4.1.
